new_dawn_server/users/api/resources.py

```diff
@@ -279,6 +281,15 @@ class Meta:
         queryset = Profile.objects.all()
         resource_name = "profile"
 
+    def apply_filters(self, request, applicable_filters):
+        ranking = request.GET.get('ranking', False)
+        viewer_id = request.GET.get('viewer_id','')
+        filtered = super(ProfileResource, self).apply_filters(request, applicable_filters)
+        if viewer_id:
+            filtered = filtered.exclude(id=viewer_id)
```

Comment:
I think we should use "user_id"(phone number) here, since we use that to identify users now.